src/main/kotlin/calculator/FourArithmetic.kt

```diff
@@ -30,7 +29,8 @@ object FourArithmetic {
         execute.apply { return operationNumbers.firstNumber.times(operationNumbers.secondNumber) }
     }
 
-    fun div(operationNumbers: OperationNumbers): Int {
+    fun division(operationNumbers: OperationNumbers): Int {
+        if (confirmSecondNumberZero(operationNumbers)) throw ArithmeticException(ErrorMessage.NOT_ZERO_NUMBER_DIVISION)
```

Comment:
if문안에 line이 하나더라도 중괄호로 묶어주면 가독성이 더 좋아질 것 같아요!

```suggestion
        if (confirmSecondNumberZero(operationNumbers)) {
            throw ArithmeticException(ErrorMessage.NOT_ZERO_NUMBER_DIVISION)
        }
```

Reply:
꼼꼼한 피드백 감사드립니다! 🙇🏻‍♂️